source/src/in/joind/DateHelper.java

```diff
@@ -16,13 +17,14 @@
      */
     public static String parseAndFormat(String input, String format) {
         // Assume that the input format is ISO8601
-        SimpleDateFormat dfOutput = new SimpleDateFormat(format), dfInput = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+        SimpleDateFormat dfOutput = new SimpleDateFormat(format, Locale.US),
```

Comment:
Question for this and the other Locale adjustments - should this use the user's current Locale, rather than defaulting to the US? Does specifying a locale here actually make a difference to the final output?